autoload/neomake/makers/ft/vim.vim

```diff
@@ -14,8 +14,11 @@ function! neomake#makers#ft#vim#vint() abort
 
     return {
         \ 'args': l:args,
-        \ 'errorformat': '%I%f:%l:%c:style_problem:%m,%f:%l:%c:%t%*[^:]:%m'
-        \ }
+        \ 'errorformat': '%I%f:%l:%c:style_problem:%m,%f:%l:%c:%t%*[^:]:%m',
+        \ 'postprocess': {
+        \   'fn': function('neomake#postprocess#GenericLengthPostprocess'),
+        \   'pattern': '\vUndefined variable: \zs([^ ]+)',
```

Comment:
Might make sense.
Currently it is handled by the postprocess function itself, but we could make the default one accept a list, too.
OTOH, using a list for the postprocess handlers felt dirrty already; as for the pattern, it could be easily be a group/ORed pattern.